docs/user-guide/ui/access/vpn.md

```diff
@@ -8,16 +8,16 @@ If you don't know what VPN is; below you can read a little description you can f
 There are 2 major use cases for a VPN. For our use of VPN(s) with DAppNode we use the VPN to be able to securely connect to the private internal Docker network of your DAppNode, a private network that exists within your DAppNode, which is how you can access and navigate the Admin UI and all other package specific UI's that are unavailable using the Local Proxy by creating a direct virtual encrypted tunnel that makes your client device part of the internal DAppNode private network.  
-The other primary use of a VPN which yu may already know is a privacy use case where users purchase subscriptions from VPN providors who instead of connecting your tunnel to a specific private LAN they create a tunnel from your client device to their VPN server which then allows you to browse the Internet from a different goegraphic location and IP address than you would without a VPN_
+The other primary use of a VPN which you may already know is a privacy use case where users purchase subscriptions from VPN providors who instead of connecting your tunnel to a specific private LAN they create a tunnel from your client device to their VPN server which then allows you to browse the Internet from a different goegraphic location and IP address than you would without a VPN._
 
-The main purpose of setting up VPN access is to allow the user to safely connect to their DAppNode locally or remotely, in both cases these are encrypted tunnels that allow for complete access to your DAppNode. While you can use the DAppNode created WiFi hotspot to access the same things, you are limited to the range of the Node's WiFi, and internet access is generally heavily degraded since you are routing everything on your client through your node, which works, but is not designed to be used as such. This is why VPNs although just one of several ways to connect to your DAppNode, is the most important, and should be used most often, as the default method of connecting
+The main purpose of setting up VPN access is to allow the user to safely connect to their DAppNode locally or remotely, in both cases these are encrypted tunnels that allow for complete access to your DAppNode. While you can use the DAppNode created WiFi hotspot to access the same things, you are limited to the range of the Node's WiFi, and internet access is generally heavily degraded since you are routing everything on your client through your node, which works, but is not designed to be used as such. This is why VPNs although just one of several ways to connect to your DAppNode, is the most important, and should be used most often, as the default method of connecting.
```

Comment:
```suggestion
The main purpose of setting up VPN access is to allow the user to safely connect to their DAppNode locally or remotely, in both cases these are encrypted tunnels that allow for complete access to your DAppNode. While you can use the DAppNode created WiFi hotspot to access the same things, you are limited to the range of the Node's WiFi, and internet access is generally heavily degraded since you are routing everything on your client through your node, which works, but is not designed to be used as such. This is why VPN(s), although just one of several ways to connect to your DAppNode, are the most important, and should be used most often, as the default method of connecting to your DAppNode.
```